docs/migrating-to-v6.md

```diff
@@ -845,15 +822,10 @@ Here are all the specific changes that we made to the `Utils` object:
    </div><div>After
 
    ```ts
-   shopify.webhooks.addHandlers({
-     PRODUCTS_CREATE: {
-       path: '/webhooks',
-       webhookHandler: productCreateWebhookHandler,
-     },
-     PRODUCTS_DELETE: {
-       path: '/webhooks',
-       webhookHandler: productDeleteWebhookHandler},
-   });
+   shopify.webhooks.addHttpHandlers([
+     {topic: 'PRODUCTS_CREATE', handler: productCreateWebhookHandler},
+     {topic: 'PRODUCTS_DELETE', handler: productDeleteWebhookHandler},
+   ]);
 
    const topics = shopify.webhooks.getTopics();
```

Comment:
Should we name this `getHttpTopics` for consistency? Since all of the other HTTP-only methods got renamed, we should probably do it here too.